src/api_server/src/request/logger.rs

```diff
@@ -27,14 +27,14 @@ mod tests {
 
     #[test]
     fn test_parse_put_logger_request() {
-        let body = r#"{
+        let mut body = r#"{
```

Comment:
nit: in the other tests, usually we chose to redefine a variable instead of reassigning to it.